src/components/PlioQuestion.vue

```diff
@@ -258,9 +258,12 @@ export default {
             img_tag.setAttribute("id", "question_image")
             // scale the image such that width is 1/3rd of screen width
             // maintaining the aspect ratio
-            var aspectRatio = (img_tag.offsetHeight/img_tag.offsetWidth)
+            var aspectRatio = 0
+            if (img_tag.offsetHeight){
+              aspectRatio = (img_tag.offsetWidth / img_tag.offsetHeight)
+            }
             img_tag.style.width = String(window.screen.availWidth/3) + 'px'
-            img_tag.style.height = String(img_tag.offsetWidth * aspectRatio) + 'px'
+            img_tag.style.height = String(img_tag.offsetWidth / aspectRatio) + 'px'
```

Comment:
This will give you a division by zero error when the aspectRatio is 0.

Also this seems a little unnecessary. The aspect ratio calculation can be avoided and you can simply do:

```js
if (img_tag.offsetHeight) {
    img_tag.style.height = img_tag.offsetHeight
} else {
    img_tag.style.height = 0
}

Becuase you're dividing the `offsetWidth` by the `aspectRatio` and the result of that division is simply

Reply:
Uhhhhh how did i miss this

Reply:
Yo yo 

I think your calculation is wrong..

It should be 
```
img_tag.style.height = String(img_tag.style.width / aspectRatio) + 'px' # note `width` instead of offsetWidth
```

Otherwise the image will not maintain its AR

Reply:
No actually I want to scale the image such that it's width is 1/3rd of the screen width.
Otherwise it looks very small inherently

Reply:
Haan I understand. 

The width is 1/3rd, and the height should be set based on two things: the new width (`img_tag.style.width`) and the aspect ratio (`aspectRatio`).

Otherwise you're setting the height to the same as the old height, which is not what you want since image should scale proportionately

Reply:
isme bass `img_tag.style.width` should be `img_tag.offsetWidth`
offsetWidth gives an integer
so easy to divide directly